modules/docs/arrow-docs/docs/img/old-design/icon-video.svg

```diff
@@ -1,4 +1,4 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="28" height="28" viewBox="0 0 28 28">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="28" height="28" viewBox="0 0 28 28">
```

Comment:
This one actually did not have it weird...

Reply:
yes, did search & replace first. this added it a couple of places where it had not been before and also the formatting was off. i think i have it now